metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -234,16 +261,21 @@ def ping(
             properties: metadata for the event
         """
 
-        if not self.enabled or self.mp is None:
-            return
-
         if properties is None:
             properties = {}
 
         # send event
         try:
             logger.debug(f"Sending telemetry for {event_name}")
             properties.update(self._server_props(server))
+            if self.sentry_enabled:
+                from sentry_sdk import set_tag
+
+                for key in properties:
+                    set_tag(key, properties[key])
```

Comment:
Is set tag global? Do we need to unset tags after this call to prevent them from bleeding over to the next call